app/models/check.rb

```diff
@@ -98,16 +98,8 @@ def eligible_for_childcare_costs?
     ChildcareEligibilityService.call(self)
   end
 
-  def show_over_60_question?
-    !FeatureFlags.enabled?(:under_eighteen, session_data)
-  end
-
   def consolidated_client_age
```

Comment:
is this method still required? it appears to be only used once in journey_logger_service.rb can we just use client_age?

Reply:
@willc-work have updated and removed this method